forms-flow-api/tests/unit/api/test_import_support.py

```diff
@@ -597,11 +597,11 @@ def test_import_edit(app, client, session, jwt, mock_redis_client):
         response = client.post("/import", data=form_data, headers=headers)
         assert response.status_code == 200
         assert response.json is not None
-        assert response.json == {"message": "Imported successfully."}
+        assert response.json == {"formId": "66f3adcc80926673336c3c49"}
 
     # Test case 4: Import edit - only workflow
     with patch.object(ImportService, "import_form_workflow") as mock_import_service:
-        mock_response = {"message": "Imported successfully."}

```

Comment:
Only sending the formId as the response now? removed message?

Reply:
yes removed the message only formId in response